tests/forte/data/ontology/ontology_code_generator_test.py

```diff
@@ -178,6 +178,50 @@ def test_include_and_exclude_init(self):
             exp_files = [file for file in exp_files if file != corrupted_path]
             self.assertEqual(gen_files, exp_files)
 
+    def test_namespace_depth(self):
```

Comment:
maybe we could use `@data` to try different namespace depth and different expected output. Currently we only tested `depth=1` right?